### PR TITLE
Bug 1828966: Don't blacklist SecurityContextConstraints on the search page

### PR DESCRIPTION
**Pull request by spadgett**
/assign @benjaminapetersen 
/cc @jwforres

### CODE REVIEW

**Comment by spadgett** [comment]
Screenshot_2020-04-17 Search · OKD

File: frontend/packages/operator-lifecycle-manager/integration-tests/scenarios/single-installmode.scenario.ts [hunk 1]
```diff
@@ -204,7 +204,7 @@ describe('Interacting with a `OwnNamespace` install mode Operator (Prometheus)',
     expect(crudView.successMessage.getText()).toContain('example has been updated to version');
   });
 
-  it('displays Kubernetes objects associated with the `Prometheus` in its "Resources" section', async () => {
+  xit('displays Kubernetes objects associated with the `Prometheus` in its "Resources" section', async () => {
```
Comment (benjaminapetersen) [inline frontend/packages/operator-lifecycle-manager/integration-tests/scenarios/single-installmode.scenario.ts:207]: This test needs to be disabled?

Comment (spadgett) [reply frontend/packages/operator-lifecycle-manager/integration-tests/scenarios/single-installmode.scenario.ts]: > This test needs to be disabled?

@benjaminapetersen I mean to push that to #5100, removed from this PR

**Comment by benjaminapetersen** [comment]
Want to treat this as a bug?

**Pull request merged**